pylutron_caseta/pairing.py

```diff
@@ -64,7 +64,14 @@ async def async_pair(server_addr):
         None, _generate_csr_with_ssl_context
     )
 
-    cert_pem, ca_pem = await _async_generate_certificate(server_addr, ssl_context, csr)
+    try:
```

Comment:
Why not just add both of the root CAs to the ssl context to begin with? You can change it on line 253 in `_generate_signed_ssl_context` by just concatenating the two CA certs together.